docs/streams/quickstart.html

```diff
@@ -140,25 +130,36 @@
     --topic streams-file-input
 </pre>
 
+The created topic can be described with the same <b>kafka-topics</b>> tool:
 
 <pre class="brush: bash;">
-&gt; bin/kafka-console-producer.sh --broker-list localhost:9092 --topic streams-file-input < file-input.txt
+&gt; bin/kafka-topics.sh --zookeeper localhost:2181 --describe
+
+Topic:streams-file-input	PartitionCount:1	ReplicationFactor:1	Configs:
+    Topic: streams-file-input	Partition: 0	Leader: 0	Replicas: 0	Isr: 0
+</pre>
+
+And start the console producer that can then write to this topic line-by-line,
+with each line as a separate Kafka message with null key and value encoded a string to the topic (in practice,
+stream data will likely be flowing continuously into Kafka where the application will be up and running):
+
+<pre class="brush: bash;">
+&gt; bin/kafka-console-producer.sh --broker-list localhost:9092 --topic streams-file-input
 </pre>
 
-<h4><a id="quickstart_streams_process" href="#quickstart_streams_process">Step 4: Process data</a></h4>
+<h4><a id="quickstart_streams_start" href="#quickstart_streams_start">Step 4: Start the Wordcount Streams Application</a></h4>
 
 <pre class="brush: bash;">
 &gt; bin/kafka-run-class.sh org.apache.kafka.streams.examples.wordcount.WordCountDemo
 </pre>
 
 <p>
 The demo application will read from the input topic <b>streams-file-input</b>, perform the computations of the WordCount algorithm on each of the read messages,
-and continuously write its current results to the output topic <b>streams-wordcount-output</b>.
+and continuously write its current results to the auto-created output topic <b>streams-wordcount-output</b>.
```

Comment:
Do we want to leverage auto-created topics here, actually?  Given that we strongly advice users NOT to do this, perhaps the quickstart should follow this recommendation, too.  Yes, there might be one additional command we need to run, but OTOH I am concerned that users will remember what they have done in this quickstart as "best practice" for their next experiments (I'd expect many users to follow the quickstart, and then just head off doing their own experiments rather than going carefully through all our docs...)